### ISSUE TITLE
Notification when using clipboard

### ISSUE THREAD

**Issue by barteksc** [Tkko/Flutter_Pinput#73]
If there is some text stored in the clipboard, and the app is using a `Clipboard.getData('text/plain')` method, an `{app name} pasted from {source app name}` notification is shown. That means, when using the PinPut lib, every time the app is resumed, users see that notification even if the clipboard's content is not used at all. Is there any way to fix that?

![Example image from Internet](https://www.howtogeek.com/wp-content/uploads/2020/09/iphone-paste-message.png?width=1198&trim=1,1&bg-color=000&pad=1,1)

**Comment by Tkko** [comment]
`checkClipboard: false` should help

**Comment by barteksc** [comment]
Thanks, but it doesn't work because of an `OR` condition here
```
if (appLifecycleState == AppLifecycleState.resumed ||
        widget.checkClipboard) {
      _checkClipboard();
}
```

**Comment by Tkko** [comment]
Got it, I'll take care.

**Comment by Tkko** [comment]
Fixed, just uploaded new refactored version 2.0.0.
